src/Security/Group.php

```diff
@@ -47,11 +47,12 @@
  *
  * @property int $ParentID ID of parent group
  *
- * @method Group Parent() Return parent group
- * @method HasManyList Permissions() List of group permissions
- * @method HasManyList Groups() List of child groups
- * @method ManyManyList Roles() List of PermissionRoles
  * @mixin Hierarchy
+ * @method SilverStripe\ORM\HasManyList<Group> Groups()
+ * @method SilverStripe\ORM\ManyManyList<Member> Members()
+ * @method Group Parent()
+ * @method SilverStripe\ORM\HasManyList<Permission> Permissions()
+ * @method SilverStripe\ORM\ManyManyList<PermissionRole> Roles()
```

Comment:
`HasManyList` and `ManyManyList` are imported as `use` statements above (presumably specifically for the `@method` annotations) - if those use statements aren't used elsewhere, they should be removed. Otherwise, we should probably just use the short form here in the `@method` annotations.